app/views/templates/payments/PaymentsHistoryCharge.scala.html

```diff
@@ -25,32 +25,35 @@
 @(transaction: PaymentsHistoryModel)(implicit messages: Messages)
 
 @chargeType = @{
-    if(ChargeType.positiveOrNegativeChargeTypes.contains(transaction.chargeType) && transaction.amount < 0) {
-      PaymentMessageHelper.getChargeType(transaction.chargeType.value).copy(id = "repayment")
-    } else {
-      PaymentMessageHelper.getChargeType(transaction.chargeType.value)
-    }
+  if(ChargeType.positiveOrNegativeChargeTypes.contains(transaction.chargeType) && transaction.amount < 0) {
+    PaymentMessageHelper.getChargeType(transaction.chargeType.value).copy(id = "repayment")
+  } else {
+    PaymentMessageHelper.getChargeType(transaction.chargeType.value)
+  }
 }
 
 @transaction.clearedDate match {
 
   case Some(date) => {
-    <tr class="@chargeType.id">
-      <td class="nowrap" valign="top">@displayDate(date, useShortDayFormat = true)</td>
-      <td><span class="bold">@messages(s"${chargeType.title}")</span>
+    <tr class="@chargeType.id govuk-table__row">
+      <td class="govuk-table__cell nowrap" valign="top">@displayDate(date, useShortDayFormat = true)</td>
```

Comment:
Validity has an error: "The “valign” attribute on the “td” element is obsolete. Use CSS instead."

It may simply be the case that you can remove the attribute.